### PR TITLE
Fix menu pane header. Add tests for new menu header

### PR DESCRIPTION
**Pull request by ylavoie**

### CODE REVIEW

File: UI/css/ledgersmb.css [hunk 1]
```diff
@@ -73,11 +73,15 @@ table.warning {
     border: 0;
 }
 
-#company_info, #version_info, #login_info {
+#version_info {
    font-weight: bold;
    font-size: 9pt;
 }
 
+#company_info, #login_info {
```
Comment (ehuelsmann) [inline UI/css/ledgersmb.css]: If these elements are also styled in 'expanded.css', shouldn't this be moved to there as well?

File: UI/menu/expanding.css [hunk 2]
```diff
@@ -8,3 +8,28 @@ li.menu_closed > ul {
...
 }
+
+#version_info {
+    text-align: center;
+}
+
+.header_table {
+    margin-left: auto;
+    margin-right: auto;
+    border=none; border-height=0; padding-top=0; padding-bottom=0; margin-top=0; margin-bottom=0;
```
Comment (ehuelsmann) [inline UI/menu/expanding.css]: This is invalid CSS syntax, as far as I know. CSS uses a colon, not an equation sign.

File: xt/66-cucumber/11-ar/step_definitions/pageobject_steps.pl [hunk 3]
```diff
@@ -19,9 +19,6 @@
 use Test::BDD::Cucumber::StepFile;
 
 
-use Data::Dumper;
-
-
```
Comment (ehuelsmann) [inline xt/66-cucumber/11-ar/step_definitions/pageobject_steps.pl]: Even though Data::Dumper should probably not be there, it should be PR-ed in a separate PR. (11-ar has really nothing to do with this PR...)

File: xt/lib/PageObject/App/Menu.pm [hunk 4]
```diff
@@ -76,14 +76,14 @@ my %menu_path_pageobject_map = (
...
     my ($self) = @_;
 
-    my @logged_in_found =
-        $self->find_all('*contains', text => "Logged in as");
-    my @logged_into_found =
-        $self->find_all('*contains', text => "Logged into");
+    my @logged_in_company =
+        $self->find_all("//*[\@id='company_info' and string-length(normalize-space(text())) > 0]");
+    my @logged_in_login =
+        $self->find_all("//*[\@id='login_info' and string-length(normalize-space(text())) > 0]");
```
Comment (ehuelsmann) [inline xt/lib/PageObject/App/Menu.pm]: Why do you stop checking the exact text? If there's an important reason to do that, it probably deserves mention in the log message. If there's not, I'd prefer we stick to checking that the screen has *exactly* the desired text.

File: xt/66-cucumber/11-ar/step_definitions/pageobject_steps.pl (hunk 3 above)
Comment (ylavoie) [inline xt/66-cucumber/11-ar/step_definitions/pageobject_steps.pl]: Left over from other issue. Removing

File: UI/menu/expanding.css (hunk 2 above)
Comment (ylavoie) [inline UI/menu/expanding.css]: Indeed.

File: xt/lib/PageObject/App/Menu.pm (hunk 4 above)
Comment (ylavoie) [inline xt/lib/PageObject/App/Menu.pm]: Moving to trim in the server instead

File: UI/css/ledgersmb.css (hunk 1 above)
Comment (ylavoie) [inline UI/css/ledgersmb.css]: Your call. They need to be in one place. I just fixed the layout but kept the actual distribution. This should be addressed in another PR.

Comment (ehuelsmann) [inline UI/css/ledgersmb.css]: Don't quite understand the "kept distribution" part: there's a #version_info in expanding too

Comment (ylavoie) [inline UI/css/ledgersmb.css]: It is duplicated in many places, see https://github.com/ledgersmb/LedgerSMB/search?utf8=%E2%9C%93&q=version_info&type=
This should be addressed in another PR
